include/swift/AST/DiagnosticsSema.def

```diff
@@ -3177,6 +3177,8 @@ ERROR(object_literal_broken_proto,none,
 ERROR(discard_expr_outside_of_assignment,none,
       "'_' can only appear in a pattern or on the left side of an assignment",
       ())
+WARNING(discard_expr_void_result_redundant, none,
+        "ignoring the result of a Void-returning function is redundant", ())
```

Comment:
nit: The existing wording is fine, but "explicitly discarding" or "using '_' to ignore" instead of "ignoring" is a little more clear in my opinion.

Reply:
Yeah I agree. I have updated the wording.